test/src/SyncWhenReceiveEncrypted.cc

```diff
@@ -0,0 +1,503 @@
+// This file is under GNU General Public License 3.0
+// see LICENSE.txt
+
+#define UPDATE_DATA 0

```

Comment:
Looks like a clever way to have the test create/update its own data. While I appreciate the idea, do you see a better way?

Reply:
It's the easiest way to update all the keys and emails being emitted, I don't really see a better way than that